### ISSUE TITLE
Terraform setup uses 0.14.0 in github action

### ISSUE THREAD

**Issue by larsmars** [hashicorp/setup-terraform#54]
When running github actions with Setup terraform, hashicorp uses version 0.14.0 even though it is not released yet!
My cloud state bucket is now bricked!

```
# Install Terraform CLI
    - name: Setup Terraform
      uses: hashicorp/setup-terraform@v1.0.1
```

After running this github action I wanted to update the env. from dev machine. (as done before)

![image](https://user-images.githubusercontent.com/5154589/95188697-1790fc80-07cd-11eb-8dd0-0aa75271c449.png)

Version 0.14.0 is **unavailable** and not listed here:
https://www.terraform.io/downloads.html

0.14.0 Is stated as **unreleased** here:
https://github.com/hashicorp/terraform/releases

I know you can explicitly set version number for hashicorp/setup-terraform@v1. in github actions IE:

```
    - name: Setup Terraform
      uses: hashicorp/setup-terraform@v1
      with:
        terraform_version: 0.13.4
```

Regardless an **unreleased** and **unavailable** version should not be chosen as the default latest.

**Comment by alexjurkiewicz** [comment]
This was fixed in v1.1.0, which was released in July

**Comment by aeschright** [comment]
Closing as fixed.
